code/frameworks/nextjs/src/config/webpack.ts

```diff
@@ -42,6 +42,9 @@ const setupRuntimeConfig = (baseConfig: WebpackConfig, nextConfig: NextConfig):
    * In Next13, the `newNextLinkBehavior` option now defaults to truthy (still
    * `undefined` in the config), and `next/link` was engineered to opt *out*
    * of it
+   *
+   * TODO: Revisit logic as soon as the changes in https://github.com/vercel/next.js/pull/42623
```

Comment:
I tried to write the logic in a way that would work before and after that change is released. But probably a good idea to revisit, regardless.